contrib/julienschmidt/httprouter/option.go

```diff
@@ -23,7 +24,11 @@ type RouterOption func(*routerConfig)
 
 func defaults(cfg *routerConfig) {
 	cfg.analyticsRate = globalconfig.AnalyticsRate()
-	cfg.serviceName = "http.router"
+	cfg.serviceName = globalconfig.ServiceName()
+	if cfg.serviceName == "" ||
+		cfg.serviceName == tracer.DefaultServiceName {
+		cfg.serviceName = "http.router"
```

Comment:
Interesting - this is only a server-side framework right? It's not something also used for making client HTTP calls to external services?

Reply:
That's right, this is only server-side.